docs/adapters.rst

```diff
@@ -1,4 +1,21 @@
 Adapters
 --------
 
-.. automodule:: openscm.adapters
+Included adapters
+=================
+
+OpenSCM currently supports the following simple climate models using
+adapters:
+
++--------------------------------------------------------------------+-----------+----------+---------------------------------------------------------------------------------------------------------+
+| Model name                                                         | Stepwise? | Regions? | Comment                                                                                                 |
```

Comment:
I'd be tempted to get rid of the regions and stepwise comments have those on individual model pages instead